index.html

```diff
@@ -76,137 +83,139 @@ <h2>I’am a Software Developer</h2>
       </div>
     </section>
```

Comment:
- [ ] Kindly make sure to position your My Recent Rorks section on the center of the page

||      Your       ||        Figma       ||
|-|----------------------|-| ----------------------|-|
||![42](https://user-images.githubusercontent.com/79058364/167033284-8d49d17f-658c-4153-83b2-5f09ca8af57c.png) ||![41](https://user-images.githubusercontent.com/79058364/167033346-773253b3-4130-4698-b5e9-3c24199e957e.png)||